apps/src/componentLibrary/tabs/_Tab.tsx

```diff
@@ -97,19 +97,26 @@ const _Tab: React.FunctionComponent<TabsProps> = ({
   isClosable = false,
   onClose = () => {},
 }) => {
+  const [longTooltip, setLongTooltip] = useState<TooltipProps>();
   const handleClick = useCallback(() => onClick(value), [onClick, value]);
   const handleClose = useCallback(() => onClose(value), [onClose, value]);
-  const handleNativeTooltip = (e: React.MouseEvent<HTMLButtonElement>) => {
+  const handleNativeTooltip = (
+    e: React.MouseEvent<HTMLButtonElement> | React.FocusEvent<HTMLButtonElement>
+  ) => {
     const target = e.currentTarget;
     // Locate the element that contains the text (if nested inside spans or other elements).
     const textElement = target.querySelector('span') || target;
 
-    if (textElement.scrollWidth > textElement.clientWidth && !tooltip) {
-      // Set the tooltip text if overflow occurs if the tooltip prop is not passed.
-      target.title = textElement.textContent || '';
+    if (textElement.scrollWidth > textElement.clientWidth) {
```

Comment:
nit: we also need to keep ` && !tooltip` since logic here is that if there's a tooltip already - we don't implement native tooltip